README.md

```diff
@@ -125,8 +125,7 @@ are some helper scripts to make this easier (from [here](https://cloud.google.co
 
 ```
 gcloud components install docker-credential-gcr
-docker-credential-gcr configure-docker
-docker-credential-gcr gcr-login
+gcloud auth configure-docker
 ```
```

Comment:
one thing to note here is that you can also use an IAM service account and login with docker directly if there's no other need for having gcloud installed on the devices. https://cloud.google.com/container-registry/docs/advanced-authentication basically you create a service account and grant it read only permissions to the GCR storage bucket for the project and then they can login with docker cli directly using the service account json key like this `docker login -u _json_key -p "$(cat keyfile.json)" https://us.gcr.io` (assuming the key is in ./keyfile.json)

Reply:
Thanks @programatt, sorry for going so long. I've spent some time deep in Docker land in the last few weeks and think it will work well, thanks for all the tips.

We do use `gcloud` and `gsutil` pretty extensively and rely on a GCP network connection, so we do need them to authenticate at some point. We are using service account keys in a number of places and they work well.

I think I know have it so the images are public so anyone should be able to pull them, which is what I'm going for.

Reply:
@wtgee awesome! Glad to hear it seems to be going well for this project.